pkgs/top-level/darwin-packages.nix

```diff
@@ -33,8 +33,12 @@ makeScopeWithSplicing (generateSplicesForMkScope "darwin") (_: {}) (spliced: spl
   # macOS 11.0 SDK
   apple_sdk_11_0 = pkgs.callPackage ../os-specific/darwin/apple-sdk-11.0 { };
 
+  # macOS 13.3 SDK
+  apple_sdk_13_3 = pkgs.callPackage ../os-specific/darwin/apple-sdk-13 { };
+
   # Pick an SDK
-  apple_sdk = if stdenv.hostPlatform.isAarch64 then apple_sdk_11_0 else apple_sdk_10_12;
+  apple_sdk = apple_sdk_13_3;
```

Comment:
I think we should drop these changes to avoid a treewide rebuild on master.

Reply:
Whether to bump the default SDK should be a separate PR. Going to 13.3 might be a little too aggressive for users who haven’t (or can’t) update to 13.3 yet.

Reply:
Agreed, sorry that was mostly to simplify local testing